src/main/java/com/google/api/codegen/config/ConfigGeneratorApi.java

```diff
@@ -141,6 +146,53 @@ protected void process() throws Exception {
     return LanguageGenerator.generate(packageName);
   }
 
+  /**
+   * Generates a collection configurations section.
+   */
+  private static List<Object> generateCollectionConfigs(Map<String, String> nameMap) {
```

Comment:
Maybe move above generateMethodConfigs(), where the old generateCollectionConfigs used to be